File: Python/codeup_1093_이상한 출석 번호 부르기1.py
n = int(input())
tmp = input().split()

arr = []

for i in range(0, 23):
    arr.append(0)

for i in range(0, n):
    a = int(tmp[i])
    arr[a - 1] += 1    

for i in range(0, 23):
    print(arr[i], end = " ")

File: C++/codeup_1093_이상한 출석 번호 부르기1.cpp
#include<iostream>

using namespace std;

int main()
{
    int n , arr[23] = {};
    int a;

    cin >> n;

    for(int i = 0; i < n; i++)
    {
        cin >> a;
        arr[a - 1] += 1;
    }

    for(int i = 0; i < 23; i++)
    {
        cout << arr[i] << " ";
    }

    return 0;
}

